docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc

```diff
@@ -93,36 +155,35 @@ Follow these steps to finish the process task forms:
 * *inlimit*
 * *incdownpayment*
 
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
+.. Click the *process-app-start* label to return to the *Assets* view of the project.
 
-. Click the *CorrectData-taskform* form. 
+. Click the *CorrectData-taskform* form.
 .. Remove the following subform:
 
 * *taskoutputApplication*
 
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
+.. Click the *process-app-start* label to return to the *Assets* view of the project.
 
 . Click the *Qualify-taskform* process form.
 .. Next to the *inlimit* field, click the edit icon (image:gsgEditBtn.png[]), then click *Edit*.
 +
-In the *Field Properties* dialogue, change the *Label* textbox to `In limit?`. Then click *Ok*.
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+In the *Field Properties* window, change the *Label* textbox to `In limit?`. Then click *Ok*.
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
+.. Click the *process-app-start* label to return to the *Assets* view of the project.
 
 . Click the *IncreaseDownPayment-taskform* form.
 .. Next to the *incdownpayment* field, click the edit icon (image:gsgEditBtn.png[]), then click *Edit*.
 +
-In the *Field Properties* dialogue, change the *Label* textbox to `Increase Down Payment`. Then click *Ok*.
-.. Click *Save*, then click *Save* to confirm, to save your changes.
-.. Click the *process-app-start* label to return to the *Assets* view of the project. 
+In the *Field Properties* window, change the *Label* textbox to `Increase Down Payment`. Then click *Ok*.
+.. Click *Save*, then click *Save* to confirm, to confirm your changes.
+.. Click the *process-app-start* label to return to the *Assets* view of the project.
 
 . Click the *FinalApproval-taskform* form.
 .. Next to the *inlimit* field, click the edit icon (image:gsgEditBtn.png[]), then click *Edit*.
 +
-In the *Field Properties* dialogue, change the *Label* textbox to `In limit?`. Then click *Ok*.
+In the *Field Properties* window, change the *Label* textbox to `In limit?`. Then click *Ok*.
 
 .. Click *Save*, then click *Save* to confirm, to save your changes.
```

Comment:
I don't know if this needs both "to confirm" and "to save your changes"? Does "then click Save to confirm and save your changes" still mean the same thing?

Reply:
Saw that one last night. Thought that I had caught it. Thanks for the catch.